lib/ReactViews/Custom/Chart/ChartPanelDownloadButton.tsx

```diff
@@ -69,9 +69,11 @@ async function download(items: TableMixin.Instance[]) {
   const synthesized = synthesizeNameAndValueArrays(items);
 
   // Could implement this using TaskProcessor, but requires webpack magic.
-  const worker = new Worker(
-    new URL("./downloadHrefWorker.js", import.meta.url)
-  );
+  const worker = await import(
+    //@ts-expect-error: aaa
```

Comment:
I assume you know the reason you had to put this ts-expect-error in, put that instead of "aaa".

And style wise, no other ts-expect-error in terriajs has a colon after it, and most of them have a space between the `//` and `@ts-expect-error`.